<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormItemList</class>
 <widget class="QWidget" name="FormItemList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item List</string>
  </property>
  <widget class="QTableWidget" name="itemListTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>791</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Label</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Base Unit of Measure</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit Price</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter</string>
   </property>
   <widget class="QPushButton" name="btnApplyFilter">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>100</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Filter</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>691</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtCode"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtLabel"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtDescription"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Description 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtDescription2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>341</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Unit of Measure</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtUnitMeasure"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Unit Price</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtUnitPrice"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
